### DISCUSSION TITLE
Compatible with JDK8 version 2.23 of nacos

### DISCUSSION THREAD

**Discussion by cx0217** [loki4j/loki-logback-appender#256] [Q&A]
Version  jdk8 + nacos 2.2.3 + loki-logback-appender-jdk8 (1.2.0~1.5.1) having 

|-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[CONFIG_LOG_FILE] - 'File' option has the same value "C:\Users\aurora/logs/nacos/config.log" as that given for appender [CONFIG_LOG_FILE] defined earlier.
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[CONFIG_LOG_FILE] - Collisions detected with FileAppender/RollingAppender instances defined earlier. Aborting.
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[CONFIG_LOG_FILE] - For more information, please visit http://logback.qos.ch/codes.html#earlier_fa_collision
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[NAMING_LOG_FILE] - 'File' option has the same value "C:\Users\aurora/logs/nacos/naming.log" as that given for appender [NAMING_LOG_FILE] defined earlier.
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[NAMING_LOG_FILE] - Collisions detected with FileAppender/RollingAppender instances defined earlier. Aborting.
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[NAMING_LOG_FILE] - For more information, please visit http://logback.qos.ch/codes.html#earlier_fa_collision
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[REMOTE_LOG_FILE] - 'File' option has the same value "C:\Users\aurora/logs/nacos/remote.log" as that given for appender [REMOTE_LOG_FILE] defined earlier.
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[REMOTE_LOG_FILE] - Collisions detected with FileAppender/RollingAppender instances defined earlier. Aborting.
19:52:27,330 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[REMOTE_LOG_FILE] - For more information, please visit http://logback.qos.ch/codes.html#earlier_fa_collision
19:52:27,330 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[NAMING_LOG_FILE] - Attempted to append to non started appender [NAMING_LOG_FILE].
19:52:27,343 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[NAMING_LOG_FILE] - Attempted to append to non started appender [NAMING_LOG_FILE].
19:52:27,344 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[NAMING_LOG_FILE] - Attempted to append to non started appender [NAMING_LOG_FILE].
19:52:27,370 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[CONFIG_LOG_FILE] - Attempted to append to non started appender [CONFIG_LOG_FILE].
19:52:27,370 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[CONFIG_LOG_FILE] - Attempted to append to non started appender [CONFIG_LOG_FILE].
19:52:27,370 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[CONFIG_LOG_FILE] - Attempted to append to non started appender [CONFIG_LOG_FILE].
[transfer] INFO  2024-09-11 19:52:27.387 [线程号:1]  com.alibaba.nacos.plugin.auth.spi.client.ClientAuthPluginManager.init:56 - [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.impl.NacosClientAuthServiceImpl success.
[transfer] INFO  2024-09-11 19:52:27.394 [线程号:1]  com.alibaba.nacos.plugin.auth.spi.client.ClientAuthPluginManager.init:56 - [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.ram.RamClientAuthServiceImpl success.
19:52:27,410 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[REMOTE_LOG_FILE] - Attempted to append to non started appender [REMOTE_LOG_FILE].
19:52:27,442 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[REMOTE_LOG_FILE] - Attempted to append to non started appender [REMOTE_LOG_FILE].
19:52:27,442 |-WARN in ch.qos.logback.core.rolling.RollingFileAppender[REMOTE_LOG_FILE] - Attempted to append to non started appender [REMOTE_LOG_FILE].

**Comment by cx0217** [comment]
Please help solve the big guys, more urgent！

**Comment by cx0217** [comment]
base  springboot version 2.7.12

**Reply by codeLikeO** [reply]
Downgrade the Nacos-client version to 2.0.4 @cx0217
